element_calcium_imaging/field_processing.py

```diff
@@ -132,14 +133,18 @@ def make(self, key):
                 prepared_input_dir = output_dir.parent / "prepared_input"
                 prepared_input_dir.mkdir(exist_ok=True)
 
-                image_files = [
-                    PVmeta.write_single_bigtiff(
-                        plane_idx=plane_idx,
-                        channel=channel,
-                        output_dir=prepared_input_dir,
-                        caiman_compatible=True,
-                    )
-                ]
+                if multi_tiff:
```

Comment:
if it is `multi_tiff`, do we no longer need to split it into multiple files for each specific `plane_idx` and `channel`?